Developer_Documentation/1.6/code_docs/modules/shipping_modules.md

```diff
@@ -0,0 +1,217 @@
+# Introduction
+A shipping module applies a shipping cost to an order, based on some calculation method.  Each shipping module requires a minimum of two files:
+
+1. A `class` file: /includes/modules/shipping/`modulename`.php
+1. A `language` file: /includes/languages/`current_language`/modules/shipping/`modulename`.php
+
+The `language` file contains all the translatable language text for the module while the `class` file contains the module's processing portion.  A shipping module can include one or more `methods`.
+
+## Shipping Class
+The class defined in `/includes/classes/shipping.php` handles the loading of the shipping modules during Zen Cart storefront order-processing.
+
+## Anatomy of a Shipping Module
+
+### Class Names
+A shipping method's filename (e.g. `myshipping`.php) is also used as its class name:
+```php
+class myshipping extends base
+{
+  ...
+}
+```
+**Note:** Since an order's shipping selection is stored as a the underscore (_) delimited combination of a shipping module's name and the shipping-method selected, a shipping module's `name` **cannot** include an underscore!
```

Comment:
`is stored as a the` feels ambiguous. 

I suggest rewording to:
```diff
-**Note:** Since an order's shipping selection is stored as a the underscore (_) delimited combination of a shipping module's name and the shipping-method selected, a shipping module's `name` **cannot** include an underscore!
+**Note:** Since an order's shipping selection is stored as a string combining a shipping module's name plus an underscore (_) plus the shipping-method selected, a shipping module's `name` **cannot** include an underscore!
```